olive/model/config/kv_cache_config.py

```diff
@@ -70,17 +71,20 @@ def _get_v_names(self, direction="inputs"):
         else:
             return [self.ort_present_value_name.replace("<id>", str(i)) for i in range(self.num_hidden_layers)]
 
+    def _get_kv_names(self, direction="inputs"):
+        return list(chain.from_iterable(zip(self._get_k_names(direction), self._get_v_names(direction))))
+
+    def get_ort_past_kv_names(self):
+        return self._get_kv_names("inputs")
+
     def get_ort_past_key_names(self):
         return self._get_k_names("inputs")
 
     def get_ort_past_value_names(self):
         return self._get_v_names("inputs")
```

Comment:
Better to remove these two functions and update the corresponding calling part with `get_ort_past_kv_names`.

Reply:
done